util/debug/debug.go

```diff
@@ -27,3 +33,18 @@ func StackCompact(allGoroutines bool) string {
 
 	return base64.StdEncoding.EncodeToString(buf.Bytes())
 }
+
+// SaveStackToRepo collects current stack of goroutines and saves it into /logs folder inside provided directory
+func SaveStackToRepo(repoPath string, allGoroutines bool) error {
```

Comment:
Can't understand how logrotate would work

Reply:
Will be discussed and implemented under https://linear.app/anytype/issue/GO-1734/graylog-should-have-ability-to-write-the-file